cmd/jimmsrv/main.go

```
@@ -209,6 +217,17 @@ func start(ctx context.Context, s *service.Service) error {
 	})
 	s.Go(httpsrv.ListenAndServe)
 	zapctx.Info(ctx, "Successfully started JIMM server")
+	maxConccurentConncetions, _ := strconv.Atoi(os.Getenv("JIMM_SSH_MAX_CONCURRENT_CONNECTIONS"))
```

let's not ignore this error

---

i though that if you missconfigured the max connections this field will be set to 0, becoming the default value as a consequence, which is alright, considering max concurrent connections is not a "life or death" configuration value. what do you think?

---

what would a 0 value mean for max concurrent connections? would it mean nobody would be able to use ssh or that we allow infinite concurrent connections?

---

basically it will trigger this code: https://github.com/canonical/jimm/blob/c0b0b11f1dc9f5db2fc9931d93f5ae83c8800f42/internal/ssh/ssh.go#L99-L110 and we'll set the default value